apps/event-worker/src/job-execution-dispatch/github.ts

```diff
@@ -0,0 +1,76 @@
+import { Queue } from "bullmq";
+import ms from "ms";
+import pRetry from "p-retry";
+
+import { db } from "@ctrlplane/db/client";
+import { jobExecution, JobExecution } from "@ctrlplane/db/schema";
+import { Channel } from "@ctrlplane/validators/events";
+
+import { configSchema, convertStatus, getOctokit } from "../github-utils.js";
+import { redis } from "../redis.js";
+
+const jobExecutionSyncQueue = new Queue(Channel.JobExecutionSync, {
+  connection: redis,
+});
+
+export const dispatchGithubJobExecution = async (je: JobExecution) => {
+  console.log(`Dispatching job execution ${je.id}...`);
+
+  const config = je.jobAgentConfig;
+  const parsed = configSchema.safeParse(config);
+  if (!parsed.success)
+    throw new Error(`Invalid job agent config for job execution ${je.id}`);
+
+  const octokit = getOctokit();
+  if (octokit == null) throw new Error("GitHub bot not configured");
+
+  const installationToken = (await octokit.auth({
+    type: "installation",
+    installationId: config.installationId,
+  })) as { token: string };
+
+  await octokit.actions.createWorkflowDispatch({
+    owner: config.login,
+    repo: config.repo,
+    workflow_id: config.workflowId,
+    ref: "main",
```

Comment:
```suggestion
    ref: config.ref,
```
maybe this configurable?